BP/history/provinces/bp_k_tarraconensis.txt

```diff
@@ -1,13 +1,56 @@
-﻿# k_tarraconensis
+﻿#Notes:
+# Basque settlement based on toponyms:
+# https://en.wikipedia.org/wiki/Basque_language#/media/File:Euskararen_aztarna_geografikoak.svg
```

Comment:
The blue there is the maximum extension at the beginning of the common era. Is this still the case by 476?

Reply:
The other map i found was this:
![image](https://user-images.githubusercontent.com/7620180/132954949-e4b7fae4-3063-417a-864a-ef5773f747cf.png)

Reply:
Looking at your second map (and assuming I'm reading things correctly), indicates a residual Aquitanian population more or less in the Southern Pyrenees while the your first map closely matches the map I used. I presume that residual mountainous population (while low in number and therefore "residual") was probably still extant if under Iberian control around 200 BC. Your first map, along with the Toponym map I posted, seem to support an extant Aquitanian/Basque population in the regions I listed until after the Millennium.

Reply:
Ok